src/pages/Clubs/Clubs.scss

```diff
@@ -1,14 +1,12 @@
-.get-involved {
-  &__description {
-    margin-bottom: 50px;
-  }
-
+.clubs {
   &__items {
     padding-left: 0;
     list-style: none;
     columns: 2;
     
     @media screen and (max-width: $breakpoint-mobile) {
+      margin: 0 10% 0 10%;
```

Comment:
we shouldnt use 10% here, please use a standard spacing like 1rem or 2rem

also mobile style could use a second pass, this looks a bit weird to me:
<img width="357" alt="Screen Shot 2021-03-31 at 1 13 59 PM" src="https://user-images.githubusercontent.com/13306885/113205453-2825ab00-9223-11eb-9024-70df31ff58d3.png">